.env-example

```diff
@@ -3,6 +3,8 @@ VUE_APP_MS_SOCK=
 VUE_APP_MS_AUTH=
 VUE_APP_MS_MAPS=
 VUE_APP_MS_USER=
+VUE_APP_MS_WG_API=
 ROOM_PRIMARY=#00F154
 VUE_APP_DEFAULT_LOCALE=en
-VUE_APP_LOCALE=en
\ No newline at end of file
+VUE_APP_LOCALE=en
+VUE_APP_AUTH_EXPIRY_PERIOD=6
```

Comment:
Uhm? You should not configure this value...

Reply:
example file

Reply:
You should do it dynamically based on user actions, like if user leaves. With 6h expiry you cause loop.

Also the expiring time changes on different providers based their terms for the oauth.

Reply:
This cannot cause a loop. This is a check made when a user enters the app. If the token the user has expires within 6 hours of the current datetime it will go get a refresh token. If the user logs out the token is removed, if the user logs on after it has expired we log him out. How can there be a loop?

Reply:
But refresh token can expiry any date, even after 15 mins

Reply:
it expires in 30 days ?

Reply:
if it expires in a time period less than 6 hours then we just use this env variable to set it appropriately.

Reply:
It should be under 1 hour in case "remember me" token was not used and in case its remember me token then it can be 6 hours. Also some oauth providers like Google force us to have specific  maximum expiry date.

Reply:
Let's go for an hour then?

Reply:
I See there should not be anything static.